staking/programs/integrity-pool/src/lib.rs

```diff
@@ -226,18 +225,19 @@ pub mod integrity_pool {
         let token_program = &ctx.accounts.token_program;
         let publisher = &ctx.accounts.publisher;
 
-        let reward_amount_frac: frac64 = pool_data.calculate_reward(
+        // reward amount in PYTH with decimals
+        let reward_amount: frac64 = pool_data.calculate_reward(
```

Comment:
so frac64 is defined as a number with 6 decimals?

Reply:
yes. we have to treat it like float.